_lib/cms/components/Checkboxes.php

```diff
@@ -182,12 +184,11 @@ public function format_value($value, $field_name)
     }
 
     // generates sql code for use in where statement
-    public function conditions_to_sql($field_name, $value, $func = '', $table_prefix = '')
+    public function conditions_to_sql($field_name, $value, $func = '', $table_prefix = ''): string
     {
-        return null;
```

Comment:
checkboxes returns null deliberately because the search uses a table join instead.

Reply:
Could we return string or null? `:?string`

Reply:
this component should always return null as the search is handled here:
https://github.com/adamjimenez/shiftlib/blob/master/_lib/cms/cms.php#L309

This should be re-factored but we'll need to think about how we handle the joins..

Reply:
This should be fixed now